scripts/delete_merge_debug.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+"""
+Deletes all store entries with type `merge-authors-debug`.
+"""
+import argparse
+import time
+from pathlib import Path
+
+import web
+
+import infogami
+from openlibrary.config import load_config
+
+DEFAULT_CONFIG_PATH = "/opt/olsystem/etc/openlibrary.yml"
+RECORD_TYPE = "merge-authors-debug"
+
+
+def setup(config_path):
+    if not Path(config_path).exists():
+        raise FileNotFoundError(f'no config file at {config_path}')
+    load_config(config_path)
+    infogami._setup()
+
+
+def delete_records():
+    while keys := web.ctx.site.store.keys(type=RECORD_TYPE):
+        for key in keys:
+            web.ctx.site.store.delete(key)
+            time.sleep(0.5)
```

Comment:
Do we have the capacity for two store deletions a second (plus business as usual reads and writes)?